### PR TITLE
chore: a bunch of small improvements

### PR DESCRIPTION
**Pull request by geclos**

### CODE REVIEW

File: packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts [hunk 1]
```diff
@@ -64,7 +64,6 @@ export const runEvaluationJob = async (job: Job<RunEvaluationJobData>) => {
           documentUuid,
           status: finished ? 'finished' : 'running',
           ...progress,
-          completed: progress.completed < 1 ? 1 : progress.completed,
```
Comment (andresgutgon) [inline packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts:67]: Why this?

Comment (geclos) [reply packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts:67]: it's a frontend concern

Comment (andresgutgon) [reply packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts:67]: And did you handle in frontend?

Comment (geclos) [reply packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts:67]: i don't think it's particularly important to show 1/10 vs 0/10

Comment (andresgutgon) [reply packages/jobs/src/job-definitions/batchEvaluations/runEvaluationJob.ts:67]: Ok then

**Pull request merged**